### PR TITLE
Bumps in garden : ign-tools2

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/685

### CODE REVIEW

File: Formula/ignition-tools1.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class IgnitionTools < Formula
```
Comment (chapulina) [inline Formula/ignition-tools1.rb:1]: @scpeters may be better to speak about this, but I think we shouldn't touch the v1 formula. i.e. just leave `ignition-tools.rb` for that.

**Review by chapulina — COMMENTED** [review]
The `tools2` file is empty

File: Formula/ignition-tools1.rb (hunk 1 above)
Comment (methylDragon) [reply Formula/ignition-tools1.rb:1]: Hmm, let me point tools1 to the `ignition-tools1.rb` and tools2 to the other one?

Comment (methylDragon) [reply Formula/ignition-tools1.rb:1]: https://github.com/osrf/homebrew-simulation/pull/1888/commits/ef7c8336687306c4830a744772f548c1a90c4757

https://github.com/osrf/homebrew-simulation/pull/1888/commits/c953959d2121a9b38e8618cc4a3c83bc97b8a864

Comment (scpeters) [reply Formula/ignition-tools1.rb:1]: this file should be deleted

File: Aliases/ignition-tools2 [hunk 2]
```diff
@@ -1 +1 @@
-../Formula/ignition-tools.rb
```
Comment (scpeters) [inline Aliases/ignition-tools2:1]: this alias should be deleted, sine there is now an ignition-tools2 formula

File: Aliases/ignition-tools1 [hunk 3]
```diff
@@ -1 +1 @@
-../Formula/ignition-tools.rb
```
Comment (scpeters) [inline Aliases/ignition-tools1:1]: this change should be reverted

**Review by scpeters — CHANGES_REQUESTED** [review]

File: Formula/ignition-tools1.rb (hunk 1 above)
Comment (methylDragon) [reply Formula/ignition-tools1.rb]: https://github.com/osrf/homebrew-simulation/pull/1888/commits/cdfd567ea7eac97ff4dd046005076b185e05408b

File: Aliases/ignition-tools2 (hunk 2 above)
Comment (methylDragon) [reply Aliases/ignition-tools2:1]: https://github.com/osrf/homebrew-simulation/pull/1888/commits/cdfd567ea7eac97ff4dd046005076b185e05408b

File: Aliases/ignition-tools1 (hunk 3 above)
Comment (methylDragon) [reply Aliases/ignition-tools1]: https://github.com/osrf/homebrew-simulation/pull/1888/commits/cdfd567ea7eac97ff4dd046005076b185e05408b

File: Formula/ignition-tools2.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
+class IgnitionTools2 < Formula
+  desc "Entry point for ignition command-line tools"
+  homepage "https://ignitionrobotics.org"
+  url "https://github.com/ignitionrobotics/ign-tools.git", branch: "main"
+  version "1.999.999~0~20220414"
+  license "Apache-2.0"
+  head "https://github.com/ignitionrobotics/ign-tools.git", branch: "ign-tools2"
```
Comment (chapulina) [inline Formula/ignition-tools2.rb:7]: I think we can remove this line, I don't see it on other `main` branch formulae:

https://github.com/osrf/homebrew-simulation/blob/master/Formula/ignition-common5.rb

And the `ign-tools2` branch doesn't exist yet

**Review by chapulina — APPROVED** [review]
LGTM and CI passed

**Pull request merged**